pytorch_cnn_trainer/plotter.py

```diff
@@ -0,0 +1,12 @@
+import matplotlib.pyplot as plt 
+
```

Comment:
Can we cover this up with `__all__` ?

Reply:
yes I'll do it but do i have to add a line for this file in __init__.py

Reply:
Yes you should modify `init` file too.

Also include this in tests 👍 

Test can be simply to check if this function is called. I would rather prefer to return a plotter object that user can re-use to get the plot